LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/table-rows-with-zero-columns.html

```diff
@@ -28,12 +28,12 @@
   </table>
 
   <table cellspacing="10">
-    <tr data-expected-width="60" data-expected-height="40"></tr>
+    <tr data-expected-width="40" data-expected-height="40"></tr>
```

Comment:
Why are we changing this, it is in sync with upstream and we are passing all test cases - https://github.com/web-platform-tests/wpt/blob/041b78b0ea/css/css-tables/tentative/table-rows-with-zero-columns.html

Reply:
work in progress and draft ;) it was midnight last night when I pushed this. 
I think the rationale with the modification in the code this test becomes aligned with Firefox results and what Safari would do. But not fully sure yet. Still experimenting.
https://wpt.fyi/results/css/css-tables/tentative/table-rows-with-zero-columns.html?label=experimental&label=master&aligned

Reply:
I was just surprised. Ignore me! :-)